3600/3/app/controllers/user_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative '../../helpers/user_helper'
+require 'digest'
+
+class UserController < AppController
+  include UserHelper
+
+  get '/sign_up' do
+    erb :sign_up
+  end
+
+  get '/sign_in' do
+    erb :sign_in
+  end
+
+  post '/sign_up' do
+    @user = User.new(name: params['username'], email: params['email'], password: params['password'])
```

Comment:
```ruby
@user = User.create(..., password: encrypted_password)

private

def encrypted_password
  Digest::SHA1.hexdigest(params[:password])
end
```

Reply:
When I'm leaving such comments I'm expecting that I will see this code in your PR